elements/rh-code-block/demo/actions.html

```diff
@@ -0,0 +1,35 @@
+<rh-code-block>
+  <rh-code-action slot="actions" action="copy">Copy to Clipboard</rh-code-action>
+  <rh-code-action slot="actions" action="wrap">
+    Toggle
+    <span slot="inactive">line wrap</span>
+    <span slot="active">overflow</span>
+  </rh-code-action>
+<script type="text/css">#content {
+  display: block;
+  background-color: var(--rh-color-surface-lighter, #f2f2f2);
+  border: var(--rh-border-width-sm, 1px) solid var(--rh-color-border-subtle-on-light, #c7c7c7);
+  border-block-start-width: var( --rh-code-block-border-block-start-width, var(--rh-border-width-sm, 1px));
+  font-family: var(--rh-font-family-code, RedHatMono, "Red Hat Mono", "Courier New", Courier, monospace);
+  color: var(--rh-color-text-primary-on-light, #151515);
+  padding: var(--rh-space-xl, 24px);
+  height: calc(100% - 2 * var(--rh-space-xl, 24px));
+  border-radius: var(--rh-border-radius-default, 3px);
+  max-width: 1000px;
+  max-height: 640px;
+  overflow-y: auto;
+}</script>
+</rh-code-block>
+
+
+<script type="module">
+  import '@rhds/elements/rh-button/rh-button.js';
+  import '@rhds/elements/rh-code-block/rh-code-block.js';
+  import '@rhds/elements/rh-tooltip/rh-tooltip.js';
+</script>
+
+<style>
```

Comment:
Action buttons aren't visible in playground demo

Reply:
this is because (at the moment) the playgrounds load up element definitions from unpkg, and this branch hasn't yet been published.  We can and should investigate loading the elements locally from `/assets/packages/...` but this is non-trivial because of [playground-elements' sandbox](https://github.com/google/playground-elements?tab=readme-ov-file#sandbox-security)